src/client/components/NewMeal.js

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react';
+import './home.css';
+
+export const NewMeal = () => {
+    // controlled inputs
+    const [mealId, setMealId] = useState('');
+    const [inputs, setInputs] = useState({});
+
+    //to get new meal id
+    fetch(`http://localhost:3000/api/meals`)
+        .then((res) => res.json())
+        .then((data) => {
+            const total = data.length;
+            setMealId(total);
+        });
+
+    //multiple inputs field handled
```

Comment:
This will not work properly. 
First it is not `mealId` that name doesn't describe the data correctly. it should be the number of meals from what the code does

The `fetch` has to be inside a `useEffect`

Reply:
I just remove this part of fetching. Since i tried to set new meal id myself but it is not necessary as meal id field is auto increment.